common/templates/layouts/collection.njk

```diff
@@ -6,36 +6,54 @@
   }) }}'});
 {% endblock %}
 
-{% if period === 'week' %}
-  {% set datetime = dateRange | formatISOWeek %}
-  {% set displayDate = dateRange | formatDateRangeAsRelativeWeek %}
-{% else %}
-  {% set datetime = dateRange | first %}
-  {% set displayDate = datetime | formatDateAsRelativeDay %}
-{% endif %}
-
 {% block pageTitle %}
-  {{ t("collections::page_title_with_date", {
-    context: context,
-    date: displayDate | lower
-  }) }}
+  {% if period === 'week' %}
+    {{ t("collections::page_title_with_date", {
+      context: context,
+      date: dateRange | formatDateRangeAsRelativeWeek
+    }) }}
+  {% else %}
+    {{ t("collections::page_title_with_date", {
+      context: context,
+      date: datetime | formatDateAsRelativeDay
```

Comment:
This has caused the page title to lose the date when not a relative date. 

I do think it would have been nicer to keep some of the original logic and make use of the nunjucks filters to handle logic of how the date is output instead of doing it all in the controllers.

Doing this in controllers has caused duplication and seems to have added a bit more logic into these templates.

<kbd><img src="https://user-images.githubusercontent.com/3327997/89767261-e14f3d80-daf9-11ea-8fb8-ab2cec995f28.png"></kbd>

Reply:
I guess we could remove the date from the title? Is it strictly necessary?
Outgoing moves for -> Outgoing moves

> [...] make use of the nunjucks filters to handle logic of how the date is output instead of doing it all in the controllers.
> Doing this in controllers has caused duplication and seems to have added a bit more logic into these templates.

The template had logic even before this change as it was assigning variables and branching depending on the period.
And I agree that the logic should be encapsulated in the controller where is testable.

I can create another filter or modify the existing `appTime` macro.

Reply:
>I guess we could remove the date from the title? Is it strictly necessary?

 I'd prefer to keep it to be honest. When using things like the back button in a browser or looking at the history it would always just say the same thing. I think it's also probably actually better that the page title always shows the full date rather than the relative given this.

>I can create another filter or modify the existing appTime macro.

I think I felt like a nunjucks filter would have been a better place to encapsulate and test this logic but given this PR has been open a while and the work has been done I'm happy to leave it in the controller.

Reply:
Amended the title, added a new filter for the week, refactored the rest of the code.